src/plugins/intel_gpu/src/graph/strided_slice.cpp

```diff
@@ -114,9 +114,10 @@ std::string strided_slice_inst::to_string(strided_slice_node const& node) {
 
     json_composite strided_slice_info;
     strided_slice_info.add("input id", input.id());
-    strided_slice_info.add("begin_param id", node.get_dependency(1).id());
-    strided_slice_info.add("end_param id", node.get_dependency(2).id());
-    strided_slice_info.add("stride_param id", node.get_dependency(3).id());
+    std::vector<std::string> dependencies_info = {"begin_param id", "end_param id", "stride_param id"};
+    for (size_t i = 0; i < node.get_dependencies().size(); ++i) {
+        strided_slice_info.add(dependencies_info[i], node.get_dependency(i).id());
+    }
```

Comment:
stride param is the only optional and others are required input. Isn't it enough to check the last param exists or not and dump it if exists?

Reply:
I'm not sure but levit-128s fp16 is running only 1 param now.
stridedslice:Slice_196 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
stridedslice:Slice_201 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
stridedslice:Slice_482 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
stridedslice:Slice_487 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)

Reply:
I'm not sure but levit-128s fp16 is running only 1 param now.
stridedslice:Slice_196 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
stridedslice:Slice_201 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
stridedslice:Slice_482 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
stridedslice:Slice_487 to_string() error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)

Reply:
Ah I see if they are constant, they are set as attribute instead of actual node..